docs/UserGuide.md

```diff
@@ -334,6 +334,13 @@ Example:
 * `makeStats 1 t/g` generates gender statistics of the persons tagged to the 1st event in the event list
 * `makeStats 3 t/a` generates age statistics of the persons tagged to the 3rd event in the event list
 
+
+<div markdown="span" class="alert alert-info">
+:information_source: **Note:** Whenever a pie chart containing one slice of 100% is made, a white line will be created within the pie chart.<br><br>
```

Comment:
Perhaps this might be a little bit clearer? What do ya think

```suggestion
:information_source: **Note:** If the pie chart created only contains one single slice (100% of the pie chart), a white line the length of the pie chart's radius will also be present .<br><br>
```